source/Octopus.Client/Repositories/BuiltInPackageRepositoryRepository.cs

```diff
@@ -57,18 +69,32 @@ public PackageFromBuiltInFeedResource PushPackage(string fileName, Stream conten
             }
                 
             contents.Seek(0, SeekOrigin.Begin);
+
+            var link = repository.Link("PackageUpload");
+            object pathParameters;
+
+            // if the link doesn't contain overwritemode then we're connected to an older server, which uses the `replace` parameter  
+            if (link.Contains("overwritemode"))
```

Comment:
Should this be using `OverwriteModeLink.Link`?

Reply:
Agree.